pages/_app.js

```diff
@@ -5,17 +5,24 @@ import { uid } from "uid";
 
 export default function App({ Component, pageProps }) {
   const [shoppingItems, setShoppingItems] = useState(shoppingItemsData);
+  const [isDelete, setIsDelete] = useState();
 
   function handleCreateItem(newItem) {
     const itemWithId = { ...newItem, id: uid(), imageUrl: "placeholder.jpg" };
     setShoppingItems((prevItems) => [itemWithId, ...prevItems]);
   }
+
+  function handleDeleteItem(id) {
```

Comment:
First of all: the function is good! It definitely works and it is definitely just as good a solution as mine. Just so you can decide for yourself what you think is better, here is my comment: I would write the function like this:

```
function handleDeleteItem(id) {
   setShoppingItems(shoppingItems.filter((item) => {
      return item.id !== id;
   }));
}
```

I think I would prefer my way of writing because it is clearer and simpler for me. But there is something to be said for your way of writing. You use the previous state with `prevItems`. I only learned that recently. Your way of writing the function is particularly useful when the update of the state is based on a value that is asynchronous or not immediately available. It guarantees that you always use the latest state, e.g. when clicking very quickly one after the other. 
I think to sum up, your way of writing the function is safer, mine is simpler. Don't see it as criticism, just as a conversation / information.
🤷‍♀️😀